src/pages/organize/[orgId]/campaigns/[campId]/callassignments/[callAssId]/callers.tsx

```diff
@@ -0,0 +1,230 @@
+import { Close } from '@material-ui/icons';
+import { GetServerSideProps } from 'next';
+import {
+  Avatar,
+  Box,
+  Fade,
+  IconButton,
+  makeStyles,
+  Paper,
+  TextField,
+  Tooltip,
+  Typography,
+} from '@material-ui/core';
+import { DataGridPro, GridColDef, GridRowData } from '@mui/x-data-grid-pro';
+import { FormattedMessage as Msg, useIntl } from 'react-intl';
+import { useEffect, useState } from 'react';
+
+import CallAssignmentLayout from 'features/callAssignments/layout/CallAssignmentLayout';
+import CallAssignmentModel from 'features/callAssignments/models/CallAssignmentModel';
+import { PageWithLayout } from 'utils/types';
+import { scaffold } from 'utils/next';
+import TagChip from 'features/tags/components/TagManager/components/TagChip';
+import useModel from 'core/useModel';
+import { ZetkinTag } from 'utils/types/zetkin';
+import ZUIResponsiveContainer from 'zui/ZUIResponsiveContainer';
+
+const useStyles = makeStyles((theme) => ({
+  chip: {
+    borderColor: theme.palette.grey[500],
+    borderRadius: '1em',
+    borderWidth: '1px',
+    color: theme.palette.text.secondary,
+    cursor: 'default',
+    display: 'flex',
+    lineHeight: 'normal',
+    marginRight: '0.1em',
+    overflow: 'hidden',
+    padding: '0.2em 0.7em',
+    textOverflow: 'ellipsis',
+  },
+}));
+
+export const getServerSideProps: GetServerSideProps = scaffold(
+  async (ctx) => {
+    const { orgId, campId, callAssId } = ctx.params!;
+
+    return {
+      props: {
+        assignmentId: callAssId,
+        campId,
+        orgId,
+      },
+    };
+  },
+  {
+    authLevelRequired: 2,
+    localeScope: [
+      'misc.breadcrumbs',
+      'layout.organize.callAssignment',
+      'pages.organizeCallAssignment',
+    ],
+  }
+);
+
+interface AssignmentPageProps {
+  assignmentId: string;
+  campId: string;
+  orgId: string;
+}
+
+const TagsCell = ({ tags }: { tags: ZetkinTag[] }) => {
+  const classes = useStyles();
+
+  return (
+    <ZUIResponsiveContainer ssrWidth={200}>
+      {(width) => {
+        const maxTags = Math.floor(width / 100);
+        const displayedTags = tags.slice(0, maxTags);
+        const hiddenTags = tags.slice(maxTags);
+
+        const tooltipTitle = hiddenTags.map((tag) => tag.title).join(', ');
+
+        return (
+          <Box alignItems="center" display="flex" width="100%">
+            {displayedTags.map((tag) => (
+              <TagChip key={tag.id} tag={tag} />
+            ))}
+            {hiddenTags.length > 0 && (
+              <Tooltip title={tooltipTitle}>
+                <Box border={2} className={classes.chip}>
+                  {`${displayedTags.length > 0 ? '+' : ''}${hiddenTags.length}`}
+                </Box>
+              </Tooltip>
+            )}
+          </Box>
+        );
+      }}
+    </ZUIResponsiveContainer>
+  );
+};
+
+const AssignmentPage: PageWithLayout<AssignmentPageProps> = ({
```

Comment:
Name of page (and props interface) seem copied from another page. Maybe call `CallersPage` or something?